packages/components/src/internal/components/forms/detail/DetailDisplay.tsx

```diff
@@ -294,7 +294,10 @@ export function resolveDetailEditRenderer(
         }
         let value = resolveDetailFieldValue(data);
 
-        const ColumnInputRenderer = resolveInputRenderer(col);
+        const ColumnInputRenderer =
+            col.conceptURI === NON_NEGATIVE_NUMBER_CONCEPT_URI
+                ? null /* Use standard forminput for media amount/unit */
```

Comment:
based on the comment, it sounds like the media case is the one that uses `NON_NEGATIVE_NUMBER_CONCEPT_URI` is that right? I would have expected that conceptURI to be used for all amounts/units